### ISSUE TITLE
missing *_probs.pbl files

### ISSUE THREAD

**Issue by mjfi2sb3** [Gaius-Augustus/BRAKER#299]
Hi,
I am training Augustus & GeneMark using BRAKER2 to use in MAKER to annotate a plant genome. However, when I run maker, I get an error that some *_probs.pbl files are missing. When I check pre-existing species in augustus, I see the following files which are not created in the BRAKER2 training

.../config/species/maize/maize_exon_probs.pbl
.../config/species/maize/maize_igenic_probs.pbl
.../config/species/maize/maize_intron_probs.pbl

Am I missing something in the parameters?

Thanks,
/sb

**Comment by tomasbruna** [comment]
Hello @mjfi2sb3,

these files should be created during a normal BRAKER2 run. Are these files created when you run some of the [test examples](https://github.com/Gaius-Augustus/BRAKER#example-data), for example [test2.sh](https://github.com/Gaius-Augustus/BRAKER/blob/master/example/tests/test2.sh)?

Best,
Tomas

**Comment by KatharinaHoff** [comment]
The files are created when etraining is called. Within braker. Check in the
logs and errors what happens with etraining.

**Comment by mjfi2sb3** [comment]
Dear Katharina,

Thanks very much for your input. I have in fact run the test scripts & they successfully generated the *pbl files. So, I looked again at the logs, I spotted an error which I thought I fixed. I basically needed to make sure that the latest filterGenesIn_mRNAname.pl  was being used.

` Training gene file in genbank format ../augustus_training/braker2/braker_est_prot/train.f.gb does not contain any training genes. Possible known causes:
(a) The AUGUSTUS script filterGenesIn_mRNAname.pl is not up-to-date with this version of BRAKER. To solve this issue, either get the latest AUGUSTUS from its master branch with
    git clone git@github.com:Gaius-Augustus/Augustus.git
or download the latest version of filterGenesIn_mRNAname.pl from https://github.com/Gaius-Augustus/Augustus/blob/master/scripts/filterGenesIn_mRNAname.pl and replace the old script in your AUGUSTUS installation folder.
 (b) No training genes with sufficient extrinsic evidence support or of sufficient length were produced by GeneMark-EX. If you think this is the cause for your problem, consider running BRAKER with different evidence or without any evidence (--esmode) for training.`
